### DISCUSSION TITLE
Right To Left (RTL) layout support

### DISCUSSION THREAD

**Discussion by WiseFerret** [zen-browser/desktop#3944] [Ideas]
7% of world population use RTL languages: Arabic, Farsi, Urdu, Amharic and more,

 Zen Browser should have RTL layout support based on the system interface language, like other browsers have. Right now we can move the tabs bar to the right using about:config (set zen.tabs.vertical.right-side to True), but that's about it. 

 A full RTL layout support includes, but is not limited to, the following:
 1. Mirroring the main layout: sidebars, main window, etc.
 2. Mirroring the  menues: sub-menues should open to the left, not to the right.
 3. Mirroring minor elements, such as the location of "x" buttons on tabs, the direction and location of arrows on menue items, etc.
 4. Mirroring the alignment and directionality of all UI strings.
 5. Mirroring specific icons, such as the "back" button.
 
Then, the work on RTL languages translations can begin. I will be happy to participate in the work on a Hebrew translation.

**Comment by ghost** [comment]
Even when people aren't speaking RTL languages, having the tabs on the right is useful. Some websites like claude.ai have their own sidebars on the left which contradict with zen's.
